pom.xml

```diff
@@ -203,6 +212,45 @@
             <version>2.11.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>kafka</artifactId>
+            <version>${test-containers.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8</artifactId>
```

Comment:
Do we still need the JRE8 Wiremock or can we move to the latest version that uses Java 11?

Reply:
As far as I can tell wiremock has java 7 dependency and then java 8+ from [the documentation](https://wiremock.org/docs/download-and-installation/).